lib/cocoapods/installer/pods_project_integrator/pod_target_integrator.rb

```diff
@@ -47,7 +47,7 @@ def inspect
       def add_copy_resources_script_phase(native_target)
         test_type = target.test_type_for_product_type(native_target.symbol_type)
         script_path = "${PODS_ROOT}/#{target.copy_resources_script_path_for_test_type(test_type).relative_path_from(target.sandbox.root)}"
-        resource_paths = [target, *target.test_dependent_targets].flat_map(&:resource_paths)
+        resource_paths = target.all_test_dependent_targets.flat_map(&:resource_paths)
```

Comment:
This now properly fixes tracking of recursive dependencies for input output path script for test targets. It was wrong and it did not include transitive dependencies.